docs/index.rst

```diff
@@ -8,20 +8,36 @@ implementations using the reactive pattern.
 Overview
 --------
 
-The pattern is "reactive" because you use :func:`@when <charms.reactive.decorators.when>`
-and similar decorators to indicate that blocks of code react to certain conditions,
-such as a relation reaching a specific `state`, a file changing, certain config
-values being set, etc.  More importantly, you can react to not just individual
-conditions, but meaningful combinations of conditions that can span multiple hook
-invocations, in a natural way.
+Juju is an open source tool for modelling a connected set of applications in a
+way that allows for that model to be deployed repeatably and consistently
+across different clouds and substrates.  Juju Charms model the individual
+applications, their configuration, and the relations between them and other
+applications.
+
+In order for the charm to know what actions to take, Juju informs it of
+life-cycle events in the form of hooks.  These hooks inform the charm of things
```

Comment:
I would argue yes, if only because it's an assumption that the docs will only be read digitally rendered. One space after a period is [best practice](http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period), like spaces instead of tabs and vim instead of emacs. Okay, kidding about that last one. Kind of.

I don't think the spaces are a blocker. I think there's some doc cleanup and standardization that needs doing in general. That's currently tracked in #114.